webapp-django/crashstats/signature/views.py

```diff
@@ -571,17 +566,16 @@ def _transform_exploitability_summary(facets):
 
 @pass_validated_params
 def signature_bugzilla(request, params):
-    '''Return a list of associated bugs. '''
+    """Return a list of associated bugs"""
     context = {}
 
     signature = params['signature'][0]
     context['signature'] = signature
-
-    bugs_api = models.Bugs()
-    context['bugs'] = bugs_api.get(
-        signatures=[signature]
-    )['hits']
-
-    context['bugs'].sort(key=lambda x: x['id'], reverse=True)
+    context['bugs'] = list(
```

Comment:
The signature report wants bugs and related bugs, so we use `.get_bugs_and_related_bugs()`.